astore/client/commands/commands.go

```diff
@@ -21,6 +21,9 @@ type Root struct {
 	*client.BaseFlags
 
 	store *client.ServerFlags
+	outputFormat string
+	outputFile string
+	outputWriter *os.File
```

Comment:
use `io.Writer` instead of `os.File`. os.File is a file on the filesystem, `io.Writer` is anything you can write into. io.Writer is accepted by Fprintf and pretty much the entire standard library.

Reply:
outputWriter is no longer needed